src/write_gcm.c

```diff
@@ -3236,6 +3250,22 @@ static int wg_json_CreateTimeSeries(
       }
     }
 
+    if (head->values[0].ds_type == DS_TYPE_ABSOLUTE) {
```

Comment:
Why did you move this? These checks are independent of the metadata values.

Reply:
This was an artifact from prior changes, I restored the original ordering.